charts/argo-cd/values.yaml

```diff
@@ -2175,9 +2175,9 @@ notifications:
   # -- Configures notification services
   # @default -- See [values.yaml]
   ## For more information: https://argocd-notifications.readthedocs.io/en/stable/services/overview/
-  notifiers:
-    service.slack: |
-      token: $slack-token
+  notifiers: {}
+    # service.slack: |
+    #   token: $slack-token
```

Comment:
Hmmm I am unsure if we should do that since this is kind of a breaking change.
We unset (via `~`) it in our values like this:

```yaml
notifications:
  notifiers:
    service.slack: ~
    service.telegram: |
      token: $telegram-token
```

Opinions @mbevc1 @davidkarlsen @oliverbaehler ?

Reply:
Hm, I reckon it shouldn't be here in the first place (also comparing to other Charts). But I do see the point of perhaps breaking some folks deployments now. I'm happy we unset first and remove at some point later.

Reply:
`service.slack` is not required, if I not overwrite this value, it always be generated in the configmap

Reply:
Yeah but for users using slack, they need to configure it after your change. Before the change, the config was built-in.

Maybe we bump the minor version and put something into the Upgrading section that we removed this?

Reply:
The minor version is updated